src/components/Calculator.js

```diff
@@ -1,129 +1,115 @@
-import React from 'react';
+import useState from 'react-hook-use-state';
 
 import '../App.css';
 
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: null,
-      next: null,
-      operation: null,
-    };
-    this.onClick = this.onClick.bind(this);
-  }
-
-  onClick = (buttonName) => {
-    this.setState((prevState) => calculate(prevState, buttonName));
-  };
-
-  render() {
-    function display(total, operation, next) {
-      if (next !== null && operation === null) {
-        return next;
-      }
-      if (operation !== null && next !== null) {
-        return next;
-      }
+function Calculator() {
```

Comment:
- [ ] Kindly refactor the code from using the `function` keyword to an arrow function as the project requirements dictate. Note that it's so important at this milestone to practice the ES6 features to create react components 😄 🙏

Reply:
@erezfree29 Please let work on the issue mentioned by the previous reviewer.